tests/serve/test_sglang.py

```diff
@@ -0,0 +1,171 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+
+import pytest
+import requests
+
+from tests.utils.managed_process import ManagedProcess
+
+logger = logging.getLogger(__name__)
+
+
+class SGLangProcess(ManagedProcess):
+    """Simple process manager for sglang shell scripts"""
+
+    def __init__(self, script_name, request, port=8000):
+        sglang_dir = "/workspace/examples/sglang"
+        script_path = os.path.join(sglang_dir, "launch", script_name)
+
+        # Verify script exists
+        if not os.path.exists(script_path):
+            raise FileNotFoundError(f"SGLang script not found: {script_path}")
+
+        # Make script executable and run it
+        os.chmod(script_path, 0o755)
+        command = ["bash", script_path]
+
+        self.port = port
```

Comment:
will want to update the bash script to take port as arg - or hard code the port and remove the argument form init